src/Files.App/Data/Models/ItemViewModel.cs

```diff
@@ -1092,7 +1117,7 @@ public async Task LoadExtendedItemPropertiesAsync(ListedItem item)
 						}
 
 						cts.Token.ThrowIfCancellationRequested();
-						_ = LoadItemThumbnailAsync(item);
+						_ = LoadBasicIconAsync(item);
```

Comment:
```suggestion
						await LoadBasicIconAsync(item);
```
It is better to await if the process is to be completed quickly,